cmd/kubeadm/app/util/config/common.go

```diff
@@ -36,20 +36,21 @@ import (
 
 	kubeadmapi "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm"
 	kubeadmscheme "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/scheme"
-	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
+	kubeadmapiv1old "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
+	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta4"
```

Comment:
Is it better to keep the alias consistent with other packages?
```suggestion
	kubeadmapiv1 "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta3"
	kubeadmapiv1latest "k8s.io/kubernetes/cmd/kubeadm/app/apis/kubeadm/v1beta4"
```

Reply:
i think we are using `kubeadmapiv1old` and `kubeadmapiv1` in all places.

Reply:
/lgtm